### PR TITLE
Analysis descriptions

### PR DESCRIPTION
**Pull request by marinzg**
Please check the description in frontend/src/app/main/home/home.component.html since that is what's going into the documentation.

### CODE REVIEW

**Comment by laky55555** [comment]
Looks good, just few little things:
1. **Descriptions** are not in the same level
image
1. In **Path length**, better description would be _The average distance between any two persons in a graph._
1. **Density description** is not completely correct for subscription graph. In subscription graph density is 1 if every person is subscripted to all analyzing channels
image

**Review by laky55555 — CHANGES_REQUESTED** [review]
Looks very nice. Can you just take a look over few changes I put

File: frontend/src/app/main/home/home.component.css [hunk 1]
```diff
@@ -50,3 +50,14 @@
 .error-message{
   color:red !important;
 }
+
+.feature {
```
Comment (marinzg) [inline frontend/src/app/main/home/home.component.css]: Some comment.

**Review by laky55555 — APPROVED** [review]
Looked at it again, looks good :+1:

**Pull request merged**